service/src/main/resources/quips/complibot/compliments.json

```diff
@@ -537,5 +548,115 @@
     "tags":[],
     "targetableText":null,
     "targetableSsml":null
+  },
+  {
+    "quipGroup":"SMOKING_HOT",
+    "text":"I'm going to have to ask you to step outside. This is a non-smoking area. And you're smoking hot.",
+    "ssml":"I'm going to have to ask you to step outside. This is a non-smoking area. And you're smoking hot.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":null,
+    "targetableSsml":null
+  },
+  {
+    "quipGroup":"OVERCLOCK",
```

Comment:
I don't remember banning nerd humor. I think we decided not to go with super technical or super niche references though. We should look at what we excluded before and reconsider. The audience who have Echos AND know how to activate third-party skills are not common users.